BUILD.bazel

```diff
@@ -38,21 +38,18 @@ package_group(
 
 pkg_npm(
     name = "npm_package",
-    package_name = "@angular/dev-infra-private",
+    package_name = "@angular/build-tooling",
```

Comment:
We need to get signoff from @jelbourn, @alxhub and @dgp1130 on naming this.

Also, would it make sense to `_` prefix?  `@angular/_build-tooling`

Reply:
I thought we were good on this, remembering our discussion in the sync:

1. These names are never going on NPM
2. The repositories exposing these internal packages are explicitly named to contain `-private-`
3. If e.g. CLI ever intends to use this, we can just change the name, but for now we use this locally through snapshot repos and want it to be short and concise

Reply:
Yeah, that was the feedback @jelbourn just gave me as well.